src/pathfinding_igvc/src/OccupancyGridResizeService.cpp

```diff
@@ -0,0 +1,52 @@
+/*
+ * Created By: Min Gyo Kim
+ * Created On: May 14th 2018
+ * Description: Implementation of occupancy grid resize service
+ */
+
+#include <OccupancyGridResizeService.h>
+
+void OccupancyGridResizeService::resizeOccupancyGridToFitGoal(
+nav_msgs::OccupancyGrid& grid, AStar::GridPoint goal) {
```

Comment:
We should add some comments to this function

Reply:
probably going to rewrite this implementation based on earlier comment